www-client/chromium/chromium-90.0.4430.212.ebuild

```diff
@@ -15,13 +15,20 @@ DESCRIPTION="Open-source version of Google Chrome web browser"
 HOMEPAGE="https://chromium.org/"
 PATCHSET="7"
 PATCHSET_NAME="chromium-$(ver_cut 1)-patchset-${PATCHSET}"
+PPC64LE_COMMIT="3f575325dcc3bdfc419824518bac6d4c38241859"
+PPC64LE_URI="https://raw.githubusercontent.com/void-linux/void-packages/${PPC64LE_COMMIT}/srcpkgs/chromium/patches"
 SRC_URI="https://commondatastorage.googleapis.com/chromium-browser-official/${P}.tar.xz
 	https://files.pythonhosted.org/packages/ed/7b/bbf89ca71e722b7f9464ebffe4b5ee20a9e5c9a555a56e2d3914bb9119a6/setuptools-44.1.0.zip
-	https://github.com/stha09/chromium-patches/releases/download/${PATCHSET_NAME}/${PATCHSET_NAME}.tar.xz"
+	https://github.com/stha09/chromium-patches/releases/download/${PATCHSET_NAME}/${PATCHSET_NAME}.tar.xz
+	ppc64? (
+		${PPC64LE_URI}/xxx-ppc64le-libvpx.patch -> ${PN}-ppc64le-${PPC64LE_COMMIT}-libvpx.patch
```

Comment:
The default `src_unpack` function will emit a warning about an unrecognized archive. I have heard rumblings that this may become an error in some future EAPI.

We should probably define our own `src_unpack` that unpacks the tarballs and skips the patch files.

Reply:
Why not bundle the patches in a tarball and host it on d.g.o? If the changes to patches are minimal we could also try to adjust them for dev and beta releases (beta changes are minimal usually).

Reply:
I can do tarball, just did it the lazy way =)

patches are not minimal and usually need basic rebases between major versions, like adjusting line numbers, handling file moves and adjusting context lines.

but if it applies it usually works.

Reply:
pushed tarred patches